pom.xml

```
@@ -144,6 +144,17 @@
             <artifactId>spring-expression</artifactId>
             <version>${spring-expression.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>${spring-expression.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-jcl</artifactId>
+            <version>5.3.39-spring-framework-5.3.45</version>
```

rename and use the variable
```suggestion
            <version>${springframework.version}</version>
```